python/generate/.openapi-generator-ignore

```diff
@@ -6,4 +6,6 @@ lusid_README.md
 test-requirements.txt
 tox.ini
 README.md
-requirements.txt
\ No newline at end of file
+requirements.txt
+setup.cfg
+.gitlab-ci.yml
```

Comment:
What are these new entries?

Reply:
new files that get created that we don't use/need